packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -181,7 +172,7 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
       styles,
       theme
     } = this.props;
-    const columnReorderProps = this._columnReorderProps;
+    const { columnReorderProps } = this.state;

```

Comment:
Nit: you can add this to the destructuring below it as its already pulling from `this.state`

Reply:
Oops, didn't see that. Do I need to make that change and commit? Or is that something you can resolve when you merge?

Reply:
change and push to this branch